esp-hal-common/src/rtc/esp32s2.rs

```diff
@@ -0,0 +1,3 @@
+pub(crate) fn init() {}
```

Comment:
I opted for the separate dummy implementation because each chip provides a specific implementation for the RTC initialization.

Having the ESP32-C3 implementation on the common source file would probably end up in it being moved out later.
I believe this way it also makes the change history a bit cleaner.

Reply:
Thinking a bit more into this, another option could be to remove the dummy files and just configure the function call to just `esp32c3` (in fact this might be what you meant)